src/usePopper.js

```diff
@@ -60,68 +132,43 @@ export default function usePopper(
     scheduleUpdate();
   }, [state.placement, scheduleUpdate]);
 
-  /** Toggle Events */
   useEffect(() => {
-    if (popperInstanceRef.current) {
-      // eslint-disable-next-line no-unused-expressions
-      eventsEnabled
-        ? popperInstanceRef.current.enableEventListeners()
-        : popperInstanceRef.current.disableEventListeners();
-    }
-  }, [eventsEnabled]);
+    if (!popperInstanceRef.current || !enabled) return;
+
+    popperInstanceRef.current.setOptions({
+      placement,
+      strategy,
+      modifiers: [...modifiers, updateModifier],
+    });
+    // intentionally NOT re-running on new modifiers
```

Comment:
why not?

Reply:
its very hard to get the memo deps right

Reply:
like you'd have to require users memoize `modifiers` i think